app/controllers/shelters_controller.rb

```diff
@@ -26,6 +26,7 @@ def create
 
       if draft.save
         redirect_to draft, notice: 'Your new shelter is pending approval.'
+        send_slack_notification "New shelter added: https://irma-api.herokuapp.com/drafts/#{draft.id}"
```

Comment:
If you use `draft_url(draft)`, that should generate a URL that works in all environments, not just production.